src/main/java/com/ldtteam/blockui/controls/EntityIcon.java

```diff
@@ -84,13 +84,19 @@ public void setEntity(@NotNull EntityType<?> type)
     public void setEntity(@NotNull Entity entity)
     {
         this.entity = entity;
-        this.setHoverPane(null);
+        if (onHover instanceof Tooltip)

```

Comment:
This is 1.18 = java with instanceof pattern matching

Reply:
You said to do it like `ItemIcon`, so I did it like that...